app/src/main/res/layout/nearby_card_view.xml

```diff
@@ -0,0 +1,96 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="?attr/mainCardBackground"
+    android:layout_centerVertical="true"
+    android:minHeight="72dp">
+
+        <Button
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:id="@+id/permission_request_button"
+            android:text="Tab to see closest wikidata item needs photo"
```

Comment:
This needs to be removed and replaced with the string below. :)

`<string name="display_nearby_notification_summary">Tap here to see the nearest place that needs pictures</string>`